language-design.md

```diff
@@ -80,4 +173,23 @@ you do so? If not, why not?
 
 **Response**
 
+I think they are fundamentally incompatible in some ways; obviously there cannot be a programming 
+language that resembles natural language but does not have natural language syntax. However, the 
+reverse is not necessarily false: you certainly can create a language that has "natural-language 
+replacements for some of the more abstruse syntax" [Pavlus, 2012] that doesn't "resemble natural 
+languages" [Cook, 2007] A lot of the syntactic sugar added to languages in recent years (for-loop 
+variations in JavaScript, f-strings in Python, etc.) utilizes natural language syntax to create a 
+memorable way to write something that is already possible (but ugly to write) in the language. This 
+definitely doesn't make a program read like a natural language, but it sure lowers the barriers to 
+entry.
+
+I would choose to include some natural language in the design of a DSL, but probably about the same 
+amount as is present in general-purpose languages like Java and Python. However, it depends on the 
+domain- Gherkin, which I explored on the last homework, was almost entirely natural language, which 
+was important as the language was intended to be read by less technically-minded people. In 
+general, however, I'm wary of natural language: the more natural language-like things are, the 
+better people *think* they understand them; whether they actually do is often the opposite. Natural 
+language carries a lot of ambivalence and connotations with it, which may not be universal; by 
+separating a program from natural language, a lot of confusion can be avoided.
```

Comment:
I think you make an excellent point about the perception of understanding a natural language vs actually understanding it. Literature is a whole field thinking about different ways to interpret language, whereas a computer only has one way.